content/en/infrastructure/storage_management/amazon_s3.md

```diff
@@ -403,19 +403,38 @@ For each bucket you want to monitor:
 ### Validation
 
 To verify your setup:
-- Wait for the first inventory report to generate (up to 24 hours for daily inventories).
-- Navigate to **Infrastructure** > [**Storage Management**][3] to see if the bucket(s) you configured are showing in the explorer list when "Monitored buckets" is selected.
+1. Wait for the first inventory report to generate (up to 24 hours for daily inventories).
+2. Navigate to **Infrastructure** > [**Storage Management**][3] to see if the bucket(s) you configured are showing in the explorer list when "Monitored buckets" is selected.
 
   {{< img src="infrastructure/storage_management/monitored-buckets.png" alt="Validate bucket is enabled for monitoring" responsive="true">}}
 
 ### Troubleshooting
 
 If you don't see data for buckets you set up for Storage Management:
-   - Check the destination bucket for inventory files. If there are no inventories, ensure the source buckets have permission to write to the destination bucket.
-   - Check the AWS integration page to confirm you're not missing any S3-related permissions as part of [Resource Collection][2]. Ensure all S3-related permission are granted.
-   - Confirm the Datadog integration can access the inventory files: ensure `s3:GetObject` and `s3:ListBucket` permissions for the destination buckets are set on the Datadog AWS Integration Role.
-   - If you're still encountering issues, [contact Datadog][1] with your bucket details, AWS account ID, and Datadog org name.
+   1. Check the destination bucket for inventory files that are not empty. If there are no inventories, ensure the source buckets have permission to write to the destination bucket.
+   2. Check the AWS integration page to confirm you're not missing any S3-related permissions as part of [Resource Collection][2]. Ensure all S3-related permission are granted.
+   3. Confirm the Datadog integration can access the inventory files: ensure `s3:GetObject` and `s3:ListBucket` permissions for the destination buckets are set on the Datadog AWS Integration Role.
+   4. If you're still encountering issues, [contact Datadog][1].
```

Comment:
Does the order matter for these troubleshooting instructions? If a user can try them independently, they should be in a bulleted list, not a numbered list.